manager/printermanager.go

```diff
@@ -225,7 +225,7 @@ func (pm *PrinterManager) applyDiff(diff *lib.PrinterDiff, ch chan<- lib.Printer
 			log.InfoPrinterf(diff.Printer.Name+" "+diff.Printer.GCPID, "Registered in the cloud")
 
 			if pm.gcp.CanShare() {
-				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope); err != nil {
+				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope, "USER", true); err != nil {
```

Comment:
It's true today:

https://github.com/google/cups-connector/blob/master/gcp/gcp.go#L481

we could change that but if we do we should probably make it a setting that can be toggled by admin.

Jay